### ISSUE TITLE
TabBarController

### ISSUE THREAD

**Issue by RennanAlves** [evnaz/ENSwiftSideMenu#53]
When I try to use the sidemenu with the tabBarController, the transition only occurs when I press a button (in my case, the backButton I created in the navigationBar) after i press the respective row in the tableView.

After the transition, the sideMenu don't dismiss and I need to press the menuButton or use the gestureRecognizer to dismiss him.

Is this a hierarchy problem? I'm trying to figure out but I don't know where to look.

**Comment by evnaz** [comment]
Yes, it's a hierarchy issue. What is the rootViewController in your case?

**Comment by RennanAlves** [comment]
Here is a sample of how I structured my storyboard.
![storyboard](https://cloud.githubusercontent.com/assets/7150601/6785833/ff5c6b4c-d166-11e4-8e2e-a4d6904cc874.png)

**Comment by vimalsaifudin** [comment]
Any solution for this issue ?
